src/plugins/empty-org-recycle-bin/schema.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
How about naming the directory simply `recycle-bin`?

In this case, the schema could look like this:

```json
{
  "$schema": "http://json-schema.org/draft-07/schema",
  "$id": "https://github.com/amtrack/sfdx-browserforce-plugin/src/plugins/recycle-bin/schema.json",
  "title": "RecycleBin Settings",
  "type": "object",
  "properties": {
    "emptyOrgRecycleBin": {
      "title": "Empty Org Recycle Bin",
      "description": "All items are permanently deleted. You can't undo this action.",
      "type": "boolean"
    }
  }
}
```

and a config could look like this:

```json
{
  "$schema": "../schema.json",
  "settings": {
    "recycleBin": {
      "emptyOrgRecycleBin": true
    }
  }
}
```